### ISSUE TITLE
Performance Reproducing of Swin-S

### ISSUE THREAD

**Issue by JaminFong** [SwinTransformer/Video-Swin-Transformer#5]
Hi,
Thanks for your great work.
I'm trying to reproducing the performance of Swin-S on K-400. Using the released checkpoint for evaluation, I got an 80.11% accuracy; Evaluating the Swin-S model trained by myself, I got an 80.35% accuracy (still ~0.2% worse than the paper reported one).
I wonder if anything is wrong. I doubt different validation data causes this as some videos are missing in the current K-400 dataset. My validation set contains 19,870 videos and the training set contains 239,687 videos, and how about the one you use?
Thanks a lot in advance.
Best.

**Comment by hust-nj** [comment]
Hi, thanks for your interest, we share the kinetics400 annotation here: https://github.com/SwinTransformer/storage/releases/tag/v1.0.6

**Comment by Raion-Shin** [comment]
I also trained Swin-T by myself but got only 55% top1-accuracy on K400 dataset. See https://github.com/SwinTransformer/Video-Swin-Transformer/issues/98.
